### PR TITLE
Fixed Link (https://github.com/eclipse-langium/langium-website/issues/187)

### PR DESCRIPTION
**Pull request by emilkrebs**
https://github.com/eclipse-langium/langium-website/issues/187

### CODE REVIEW

File: hugo/assets/scripts/sql/ui.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ class App extends React.Component<{}> {
         <div className="w-1/2 p-4 text-white overflow-auto">
           <h1 className="text-2xl">Langium/SQL</h1>
           <p className="pt-2">
-            This is a showcase of <a className="text-emeraldLangium" href="https://github.com/eclipse-langium/langium-sql" target="_blank">Langium/SQL</a>. The editor above
+            This is a showcase of <a className="text-emeraldLangium" href="https://github.com/langium" target="_blank">Langium/SQL</a>. The editor above
```
Comment (montymxb) [inline hugo/assets/scripts/sql/ui.tsx:87]: Quick note, this should be https://github.com/langium/langium-sql

Comment (emilkrebs) [reply hugo/assets/scripts/sql/ui.tsx]: Oh, thanks, should've checked that.

Comment (montymxb) [reply hugo/assets/scripts/sql/ui.tsx]: No worries!

**Review by montymxb — APPROVED** [review]
:octocat: link is good, approved!

**Review by msujew — COMMENTED** [review]
While you're at it, can you also fix the incorrect links for `eclipse-langium/langium-lox`? See [here](https://github.com/eclipse-langium/langium-website/blob/0a314aeb431e100a3bfa225b6c8a2d428c02485f/hugo/content/guides/scoping/class-member.md?plain=1#L26) for example.

**Comment by emilkrebs** [comment]
Fixed `eclipse-langium/langium-minilogo` and `eclipse-langium/langium-lox`

**Review by msujew — APPROVED** [review]
Thanks, looks good to me 👍

**Pull request merged**